src/main/php/util/data/Sequence.class.php

```diff
@@ -37,28 +37,8 @@ protected function __construct($elements) {
     $this->elements= $elements;
   }
 
-  /**
-   * Invoke terminal operation
-   *
-   * @param  function(): var $operation
-   * @return var
-   */
-  protected function terminal($operation) {
-    static $message= 'Underlying value is streamed and cannot be processed more than once';
-
```

Comment:
This has been completely removed - these exceptions are now handled in `TraversalOf::rewind()` and no longer shadow exceptions raised during iteration (as opposed to those raised from `rewind()`).